### ISSUE TITLE
Block Time vs Created Time

### ISSUE THREAD

**Issue by tiger5226** [lbryio/block-explorer#52]
Created time is when we find out about the transaction. Usually this is from when we read it from the mempool. However, with a full resync of chainquery this would be much sooner. In these cases the block explorer should show block time for both fields. If the gap is more than 1 hour a part make them both the blocktime value.

**Comment by tzarebczan** [comment]
This may be confusing to users (just got me!), we should get it fixed up....

**Comment by marcdeb1** [comment]
I don't think it makes sense to show Created Time, except if it's a mempool transaction.
Other explorers show a block time as Transaction time, and add a confirmation time :

![image](https://user-images.githubusercontent.com/29503179/104007478-49d54000-51a8-11eb-9c76-6af24e1cdfed.png)

So I would suggest removing Created Time and showing only Block Time, except for mempool transactions, and maybe add a confirmation time with less visibility.
